x-pack/plugins/apm/public/components/shared/Stacktrace/__test__/__snapshots__/Stackframe.test.tsx.snap

```diff
@@ -174,31 +276,69 @@ exports[`Stackframe when stackframe has source lines should render correctly 1`]
-                  <span
-                    className="c3"
-                  >
-                    line 
-                    307
-                  </span>
-                </styled.span>
+                <FormattedMessage
+                  defaultMessage="{fileName} in {functionName} at {lineNumber}"
+                  id="xpack.apm.stacktraceTab.exceptionLocationWithLineNumberMessage"
+                  values={
+                    Object {
+                      "fileName": <styled.span>
+                        server/routes.js
+                      </styled.span>,
+                      "functionName": <styled.span>
+                        &lt;anonymous&gt;
+                      </styled.span>,
+                      "lineNumber": <styled.span>
+                        <FormattedMessage
+                          defaultMessage="line {stackframeLineNumber}"
+                          description="Part of composite text: xpack.apm.stacktraceTab.exceptionLocationWithLineNumberMessage
+                  + xpack.apm.stacktraceTab.exceptionLocationWithLineNumberMessage.lineNumberText"
+                          id="xpack.apm.stacktraceTab.exceptionLocationWithLineNumberMessage.lineNumberText"
+                          values={
+                            Object {
+                              "stackframeLineNumber": 307,
+                            }
+                          }
+                        />
+                      </styled.span>,
+                    }
+                  }
+                >
+                  <styled.span>
+                    <span
+                      className="c3"
+                    >
+                      server/routes.js
+                    </span>
+                  </styled.span>
+                   in 
+                  <styled.span>
+                    <span
+                      className="c3"
+                    >
+                      &lt;anonymous&gt;
+                    </span>
+                  </styled.span>
+                   at 
+                  <styled.span>
+                    <span
+                      className="c3"
+                    >
+                      <FormattedMessage
+                        defaultMessage="line {stackframeLineNumber}"
+                        description="Part of composite text: xpack.apm.stacktraceTab.exceptionLocationWithLineNumberMessage
+                  + xpack.apm.stacktraceTab.exceptionLocationWithLineNumberMessage.lineNumberText"
+                        id="xpack.apm.stacktraceTab.exceptionLocationWithLineNumberMessage.lineNumberText"
+                        values={
+                          Object {
+                            "stackframeLineNumber": 307,
+                          }
+                        }
+                      >
+                        line 307
+                      </FormattedMessage>
+                    </span>
+                  </styled.span>
+                </FormattedMessage>
```

Comment:
If we look at the difference, we'll see that the initial value of FrameHeader remained (lines 306-325, 337 on the right).  `FormattedMessage` just appeared as a wrapper since it is used in the template.

![image](https://user-images.githubusercontent.com/11524637/50892952-c3e35a80-1410-11e9-8cc7-8bad49eedade.png)